test/verify/check-shell-landing-page

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python3 -cimport os, sys; os.execv(os.path.dirname(sys.argv[1]) + "/../common/pywrap", sys.argv)
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2024 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <https://www.gnu.org/licenses/>.
+
+import json
+
+import testlib
+
+
+@testlib.nondestructive
+class TestShellLandingPage(testlib.MachineCase):
+    def testBasic(self):
+        b = self.browser
+        m = self.machine
+
+        self.restore_file("/usr/share/cockpit/systemd/manifest.json")
+        manifest = json.loads(m.execute("cat /usr/share/cockpit/systemd/manifest.json").strip())
+
+        # Make the logs page the default page
+        manifest["menu"]["index"]["order"] = 100
+        manifest["menu"]["logs"]["order"] = 1
+        m.write("/usr/share/cockpit/systemd/manifest.json", json.dumps(manifest))
```

Comment:
Messing in /usr isn't right. Please use an override in /etc/cockpit/, like what a user would do. See https://cockpit-project.org/guide/latest/packages.html#package-manifest-override. THen you can also just write a static string, no need to invoke json parsing.